src/scripts/remove-user-from-group.ts

```diff
@@ -91,14 +92,22 @@ export default (
           demandOption: true,
         });
     },
+    // Note: do not use spread operator on args as it is not type safe
     async (args: {
       readonly clientId: string;
       readonly privateKey: string;
-      readonly organisationUrl: string;
+      readonly orgUrl: ReadonlyURL;
       readonly user: string;
       readonly group: string;
     }) => {
-      const client = oktaManageClient({ ...args }, ['users', 'groups']);
+      const client = oktaManageClient(
+        {
+          clientId: args.clientId,
+          privateKey: args.privateKey,
+          orgUrl: args.orgUrl.href,
```

Comment:
**Issue:** change `oktaManageClient` to take a ReadonlyURL instead of doing this in each command. We want to get the most value (in terms of type safety) from this change, so keep the 'good' type for as long as possible.